bridgestyle/arcgis/togeostyler.py

```diff
@@ -281,10 +280,8 @@ def processSymbolReference(symbolref, options):
             elif symbol["type"] == "CIMPolygonSymbol":
                 markerPlacementType = layer.get("markerPlacement",{}).get("type")
                 if markerPlacementType == "CIMMarkerPlacementInsidePolygon":
-                    markerPlacement = layer.get("markerPlacement", {})
-                    maxX = symbolizer.get("maxX") or symbolizer["size"] / 2
-                    maxY = symbolizer.get("maxY") or symbolizer["size"] / 2
-                    margin = proccessMarkerPlacementInsidePolygon(markerPlacement, maxX, maxY)
+                    markerPlacement = layer["markerPlacement"]
```

Comment:
I would let 
```suggestion
                    markerPlacement = layer.get("markerPlacement", {})
```
It should be more secure. But maybe this case never happen ?

Reply:
You're right